architecture/core_objects/openshift_model.adoc

```diff
@@ -448,31 +438,33 @@ annotations.
 
 Sample Project:
 
+====
+
 ----
 {
-    "kind": "Project",
-    "apiVersion": "v1beta1",
-    "metadata": {
-        "name": "hello-openshift",
-        "selfLink": "/osapi/v1beta1/projects/hello-openshift",
-        "uid": "72565b98-de1d-11e4-bab4-28d2444e470d",
-        "resourceVersion": "681",
-        "creationTimestamp": "2015-04-08T18:31:18Z",
-        "annotations": {
-            "description": "This is an example project to demonstrate OpenShift v3",
-            "displayname": "Hello OpenShift"
-        }
-    },
-    "displayName": "Hello OpenShift",
-    "spec": {
-        "finalizers": [
-            "openshift.com/project",
-            "kubernetes"
-        ]
-    },
-    "status": {
-        "phase": "Active"
+  "kind": "Project",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "hello-openshift",
+    "selfLink": "/osapi/v1/projects/hello-openshift",
+    "uid": "72565b98-de1d-11e4-bab4-28d2444e470d",
+    "resourceVersion": "681",
+    "creationTimestamp": "2015-04-08T18:31:18Z",
+    "annotations": {
+      "description": "This is an example project to demonstrate OpenShift v3",
+      "displayName": "Hello OpenShift",
+      "displayname": "Hello OpenShift"
     }
+  },
+  "spec": {
+    "finalizers": [
+      "openshift.com/project",
+      "kubernetes"
+    ]
+  },
+  "status": {
```

Comment:
delete?  (nothing we are illustrating as sample for a user to create from should have status i don't think.. we should only see status in samples of what is retrieved from the system)